modules/ossm-cr-gateway.adoc

```diff
@@ -12,61 +12,63 @@ Here is an example that illustrates the Istio gateway parameters for the `Servic
 [source,yaml]
 ----
   gateways:
-       istio-egressgateway:
-         autoscaleEnabled: false
-         autoscaleMin: 1
-         autoscaleMax: 5
-       istio-ingressgateway:
-         autoscaleEnabled: false
-         autoscaleMin: 1
-         autoscaleMax: 5
-         ior_enabled: true
+    egress:
+      enabled: true
+      runtime:
+        deployment:
+          autoScaling:
+            enabled: true
+            maxReplicas: 5
+            minReplicas: 1
+    enabled: true
+    ingress:
+      enabled: true
+      runtime:
+        deployment:
+          autoScaling:
+            enabled: true
+            maxReplicas: 5
+            minReplicas: 1
 ----
 
 
 .Istio Gateway parameters
 |===
 |Type |Parameter |Description |Values |Default value
 
-|`istio-egressgateway`
-|`autoscaleEnabled`
+|`gateway.egress`
+|`runtime.deployment.autoScaling.enabled`
 |This parameter enables/disables autoscaling.
 |`true`/`false`
 |`true`
 
 |
-|`autoscaleMin`
+|`runtime.deployment.autoScaling.minReplicas`
 |The minimum number of pods to deploy for the egress gateway based on the `autoscaleEnabled` setting.
 |A valid number of allocatable pods based on your environment's configuration.
 |`1`
 
 |
-|`autoscaleMax`
+|`runtime.deployment.autoScaling.maxReplicas`
 |The maximum number of pods to deploy for the egress gateway based on the `autoscaleEnabled` setting.
 |A valid number of allocatable pods based on your environment's configuration.
 |`5`
 
-|`istio-ingressgateway`
-|`autoscaleEnabled`
+|`gateway.ingress`
```

Comment:
`gateways.ingress.runtime.deployment.autoScaling.enabled`
`gateways.ingress.runtime.deployment.autoScaling.minReplicas`
`gateways.ingress.runtime.deployment.autoScaling.maxReplicas`

Again, gateways is plural.